my-sms/spec/factories/students.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'faker'
```

Comment:
@josh-cooper I do not think you need this require at all.  try removing it.
if you do though .. best to add it to be available in all specs
https://github.com/josh-cooper/my-sms/blob/develop/my-sms/spec/rails_helper.rb

Reply:
You're right I don't. Turns out I needed to move the calls to Faker into a block to make it work though.
Fixed in caed22a39dd60ef6306182bb45af87460a936a45